tests/virt_autotest/reboot_and_wait_up.pm

```diff
@@ -19,6 +19,7 @@ use ipmi_backend_utils;
 use base "proxymode";
 use power_action_utils 'power_action';
 use ipmi_backend_utils;
+use version_utils qw(is_sle is_opensuse);
```

Comment:
you don't use `is_sle` and as stated below you should also better not use `is_opensuse`

Reply:
removed both.